### DISCUSSION TITLE
Migration from Java to Kotlin

### DISCUSSION THREAD

**Discussion by TheRogue76** [reactwg/react-native-new-architecture#106] [Q&A] [answered]
Hi!

There is an obvious sentiment in software engineering: "If it ain't broke, don't fix it". And for all intents and purposes, i can see why the core team might not want to spend time on something that is a bit frivolous and in the end might not result in that much of a better output, but i was wondering, Would the core RN team be open if someone starts adding pull requests, migrating the java internals of RN to Kotlin?
Not a whole over the night PR with 5 mil lines changed, but smaller PRs migrating things little by little and making sure they work as they have already.

**Comment by cortinico** [comment, marked as answer]
Thanks for raising this. This is actually not related to the New Architecture per-se so you should raise this question here: github.com/react-native-community/discussions-and-proposals/
